.github/CONTRIBUTING.md

```diff
@@ -309,6 +309,17 @@ and the last true master commit is `ccc111` and your first commit is `mmm222`.
   git push -f
   ```
 
+#### How to run an app on the cloud with lightning that you're making changes
+
```

Comment:
can you add some motivation here for why you would want to use this

Reply:
PTAL!

Reply:
@akihironitta added? :otter:

Reply:
Yes! I belive so :)